src/utils/constants/index.js

```diff
@@ -19,8 +19,15 @@ export const TIC = 'tic';
 export const DEFAULT = 'default';
 export const LOGEMENT = 'logement';
 export const FAMILLE = 'famille';
-
-export const QUESTIONNAIRE_EXAMPLES = [SIMPSONS, TIC, LOGEMENT, FAMILLE];
+export const BUSINESSEXAMPLE = 'businessExample';
+
+export const QUESTIONNAIRE_EXAMPLES = [
+  SIMPSONS,
+  TIC,
+  LOGEMENT,
+  FAMILLE,
+  BUSINESSEXAMPLE,
```

Comment:
really minor thing: to keep things clean and rules about naming here: 
- use all const as .*_EXAMPLE for all
- or just use BUSINESS